tests/Tools/pdf-harness/compare-preview-pdf.py

```
@@ -0,0 +1,151 @@
+        "<!doctype html><html><head><meta charset='utf-8'>"
+        "<style>html,body{margin:0;padding:0;background:#fff;"
+        "font-family:-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,sans-serif}</style></head><body>"
+        f"<div style='width:{width}px'>"
+        + html_file.read_text()
+        + "</div></body></html>"
+    )
+    with tempfile.NamedTemporaryFile("w", suffix=".html", delete=False) as tmp:
+        tmp.write(wrapper)
+        tmp_path = tmp.name
+
+    subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)